spec/unit/lib/temporal/activity/task_processor_spec.rb

```diff
@@ -107,6 +109,8 @@
           allow(client)
             .to receive(:respond_activity_task_completed)
             .and_raise(StandardError)
+          # Skip sleeps during retries to speed up the test.
+          expect(Temporal::Client::Retryer).to receive(:sleep).at_least(1).times
```

Comment:
Also, I think for this spec you don't care if `:sleep` was called, but you do care that `:with_retries` was. Otherwise you're lurking into `Retryer`'s implementation details and this would break if you change `sleep` for something else, e.g. `Kernel.sleep`